.travis.yml

```diff
@@ -1,18 +1,35 @@
-language: java
-jdk:
-- openjdk8
-sudo: required
-dist: trusty
-install: gradle/installViaTravis.sh
-script: gradle/buildViaTravis.sh
-before_cache: gradle/prepCaches.sh
-cache:
-  directories:
-  - $HOME/.gradle
-notifications:
-  slack:
-    secure: F+OEHRZjNT9GC7zgykoyUwJRlIDdV3N1rIp+qNjN1xgag+49XenOZbquD2hYJAZnaVdZM8yx+8oTmMzeKWuSo7UiyWCTbU6i9qOzfktx0QVWPnfNH8N5WXmmn4xcg5blgpVwc5ye228Uc16/zkmCX2lxU2HOh3WWjzDgs1l2+Vge9wdaCan/Z1UG+JinSFb5m+ocMVEiTaYnv+P8fLd0xLWqyP017uLOkeEgQJcbevh7L3pawfmO/eS5F7ClYctHLAkKw/ErB/eWlA8js76ZnTLXsec7t4CeUK5DqCEmvg/Vt2Naryrp/jEgBWHBaE43M33Vh+7oszQebxmK8N7ri72g9wdPUPgVAm9WuRfj3fZl0WyjOGCHKn+wB9PG6C2lXZm0Xg9TMp3/zUhZxb26bqsHHPAXqQZfnsyE9CDAT2JAi+0QobdbPfXlAk/w6BsqisqRSZgRPW3jR16ga3kp09pvzDKG6fdxhR/SQq2mNIpJIskkvbYbq+m2olTxTGoCCnIIQEhZi3Amz79pA9r9GfL63g0ICFr5rFrtp+/A5cUDhpA1uW+AC0Myt3stkDZ+nsM7tUL/6Pa8UmH9fxGONL+TK41/Da/NlTpJYQkv+Ok0fCbALIHbpRCSR1/WR+ePnh9A67dPVUmztQ4HEYcG+Dt9i/ozCjTTaJNaPT9I46A=
-env:
-  global:
-  - secure: CpUdeXXD4JKwjpGYlU2LXusiCjDucEwTrp0KFMea8u6IFQdxs4EAYI+trYCM6CzaCuDt/JK9iCeKEjTnjMYU3Z+VutLlqXMis+n7SEx7UXMEqJZP6avvjEb3k87bPIYhH/6Sw7bgJygA7VBHSMa3xbCQBFrufxM5MZ+yGczQI4fnkTC8BpZ6HQEqf7SVUrYnH9kpcQSdQ9kpExQAZ0hh8fQlwCIsPcI4drSQeh5sLgB4MyuYqdW2643yi1edhu19vv5YSUVhyz+Jnm7R3TlFxVOxyIzVN0Fsyv3seNCSCk6ejO28dClEOIGF0nO6keJAJSMVkpzHkRjBgVu2lv6HvPYOQMAYk7cr9dwP7cq/b2TNMMwHWfR7b9+kVllRs36idyaDrZyfe4PX/mkvrVk+E4xgLkxX6BoRkNAmA1uWmACIX3bpCysY1hmxqWTzvSizI1wAS9RcBS9yb/V8Xn6alOr/rTP86uDMKa1ByXgq7G2QN8hPU+JqjnLG1ERu+/+bIUw3Yebz+H3t/NDGtQuFkQMU4cZoJmvph3ej6KGbhgpz7dTLVZJATJRjWE9MjI4AFvaTtJMzDZMIoCm6mEzmYOY1j3nutUp4VKn1vtmTac1rwCBWuPJ/FLTBrYUbYzVcTSum68rucD8ZCZkZHq27tMcs/Uh/HDuy3C/tGyNBRzI=
-  - secure: sutuutwXu3/M4eElZXWPEKKqxcu3ZJ00o7X56kPndhq8IgkNjG1l3xEBudBAhbk2V2APn9QIce0KYUJkiQTOZnIkvVEoKzpemfnzqcHdaZDXvHnTrorgkeO5mCZhKNhmF4Wq8i996LMNSRq8aH/SVkUjf8CccCdSM/WJZpdCsBEE/i4NPZLZM5oOeKZIr0S85AfRBk/O9QxDLwpIMXs24uYCasOz3P8yBKS2ZORSfcdlw0gctjT2L5L7t5qzQPwdlB2Fl88PfNZQAwNXly8+jEgGfxxd6KMtp2i6aJ0Up/Lt+RuFZYIW+JaDNKfYCqIJB5tLLcvnka4FnaTgX4SL3aHy0igj+OH20SxJX3y9/bn6ul5xMZd1GNQGUTq6Xrh4nAmr089jSGqYXZ2Xtn4WfLHkU0tiMbRt91EYR4+0iESvE+F2lO0wM8XSA7NY6F2xG3b2evATbGOe8jxzDGiElWo0zP1TVhB80UNHtG5RNpFZpTh36YXC+ShwGl9S1a5LRBRX63T0wup9nqeg9HhzUDdIi7TCq2vEvLdIbg9CTmKwD4qOcr3daWTnWdnVPmPU4zq+n4TTb24U7XCT/tddDZxxmW5eKby6VlZFijJ69clxhKzi5KyWSaQwYVW2EOH/X2tHQNsNmNyhGJVahzE+PlD2C82aoUckyrSFhRAdgM8=
+---
+os: linux
+dist: bionic
+group: stable vm_type=n1-standard-8
+language: shell
+
+jobs:
+  include:
+    - name: gradle build and test
+      language: java
+      jdk: openjdk8
+      before_cache: gradle/prepCaches.sh
+      cache:
+        directories:
+          - $HOME/.gradle
+      install: ./gradlew build
+      script: ./gradlew test
+
+    - name: docker build
+      services:
+        - docker
+      script:
+        - docker build -t compile -f Dockerfile.compile .
+        - docker build -f Dockerfile.ubuntu .
```

Comment:
This is slower than the normal gradle build/test runs. So we might want to just do it for 'deploys' to ECR